editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -189,6 +189,18 @@ pub fn adjust_gamma_properties(document_node: &DocumentNode, node_id: NodeId, _c
 	vec![LayoutGroup::Row { widgets: gamma }]
 }
 
+pub fn adjust_threshold_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let thereshold = number_widget(document_node, node_id, 1, "Threshold", NumberInput::default().min(0.00).max(1.), true);
```

Comment:
.clamp(0., 1.)

Reply:
Not sure what you mean, `.clamp()` does not exist for NumberInputs afaik

Reply:
Oh, derp. I thought this was f32 min and max functions.